packages/react/src/components/Card/CardWrap.js

```diff
@@ -76,12 +76,14 @@ const createEl = el =>
       contrast && (!color || !backgroundColor) && hoverStyle
   )
 
-const CardWrap = ({ rel, href, target, ...props }) => {
+const CardWrap = forwardRef(({ href, rel, target, ...restProps }, ref) => {
```

Comment:
why `forwardRef` is necessary? 🤔

Reply:
Good question, so we need to hook onto the outer wrap of the card for the `IntersectionObserver` to listen, and given that this a stateless component React doesn't keep a hold of the ref, so you have to use the `forwardRef` to get access to it.

Without this, if you try to pass a `ref` prop to `CardWrap` the node is always `undefined`.